specification/network/resource-manager/Microsoft.Network/stable/2018-10-01/networkWatcher.json

```diff
@@ -2669,7 +2669,11 @@
         },
         "workspaceResourceId": {
           "description": "Resource Id of the attached workspace ",
-         "type": "string"
+          "type": "string"
+        },
+        "trafficAnalyticsInterval": {
```

Comment:
Do any of the examples need to be updated?

Reply:
If it's an optional field this should not be an issue.

Reply:
>             If it's an optional field this should not be an issue.

Hi, Thanks for take care of the review.  Do I need to do anything like click the button "close Pull Request" after you approve the PR ?